cli/cmd/plugin/ui/web/tanzu-ui/src/views/management-cluster/azure/azure-mc-common/management-credential-step/ManagementCredentials.tsx

```diff
@@ -52,6 +52,7 @@ function ManagementCredentials(props: Partial<StepProps>) {
     const [connectionStatus, setConnectionStatus] = useState<CONNECTION_STATUS>(CONNECTION_STATUS.DISCONNECTED);
     const [message, setMessage] = useState('');
     const [regionLoading, setRegionLoading] = useState(false);
+    const [errorObject, setErrorObject] = useState<{ [key: string]: any }>({});
```

Comment:
nit: I would suggest `useState<{ [fieldName: string]: any }>({})`... and there may even be a specific error object (instead of `any`)

Reply:
Use `useState<{ [fieldName: string]: string }>({})`

Reply:
Also changed the error info showed in the page
<img width="636" alt="image" src="https://user-images.githubusercontent.com/106584724/185182352-ec4393fc-05c5-4c33-9814-c8a817275c71.png">